mvc/_atBindingMixin.js

```diff
@@ -7,15 +7,13 @@ define([
 	"./resolve",
 	"./sync"
 ], function(array, lang, declare, has, Stateful, resolve, sync){
-	if(has("mvc-bindings-log-api")){
-		function getLogContent(/*dojo/Stateful*/ target, /*String*/ targetProp){
+	var getLogContent = (has("mvc-bindings-log-api")) ? function (/*dojo/Stateful*/ target, /*String*/ targetProp){
```

Comment:
It looks you have an excessive white space here: `function (/*dojo/Stateful*/ target,`...